specification/_json_spec/ingest.put_geoip_database.json

```diff
@@ -7,8 +7,7 @@
     "stability": "stable",
     "visibility": "public",
     "headers": {
-      "accept": ["application/json"],
-      "content_type": ["application/json"]
+      "accept": ["application/json"]
```

Comment:
@pquentin isn't this what you fixed the other day for the python client? is there a way we can ignore this?

Reply:
Yes! I opened https://github.com/elastic/elasticsearch/pull/112581 late last week to fix this but it took a bit of time to be merged and backported to 8.15. Then the Elasticsearch build was failing over the weekend so there was no available artifact with the changes. Thankfully the Elasticsearch build passed during the night and this diff was thus removed automatically from this pull request.